test/util/PackageJson.spec.ts

```diff
@@ -0,0 +1,73 @@
+import PackageJson from '../../src/util/PackageJson';
+import { expect } from 'chai';
+import * as fs from "fs";
+import * as nodeFs from "fs-extra";
+
+describe('PackageJson generation and add modules test', function() {
+    this.timeout(20000);
+    const dir: string = 'packageJsonTest';
+
+    beforeEach(() => {	
+        if (!fs.existsSync(dir)){
+            fs.mkdirSync(dir);
+        }
+    });
+
+    afterEach(() => {
+        nodeFs.removeSync(dir);
+    });
+
+    after(() => {
+        if (fs.existsSync(dir)) {
+            nodeFs.removeSync(dir);
+        }
+    })
+
+    function verifiyDependency(module: string): any {
+        const content: any = JSON.parse(fs.readFileSync(dir + '/package.json', 'utf-8'));
+        const dependencies: any = content.dependencies;
+        return dependencies && Object.keys(dependencies).indexOf(module) >= 0 ? dependencies[module] : null;
+    }
+
+    it('should add module stringz', async () => {
```

Comment:
`stringz` LOL was this on purpose?

Reply:
lol yea, I was looking for a small package for the test